goss-testing/tests/common/goss-interface-has-ip.yaml

```diff
@@ -24,15 +24,26 @@
 # Variable set: network_interfaces
 command:
 {{range $interface := index .Vars "network_interfaces"}}
-  check_interface_ip_{{$interface}}:
-    title: Interface '{{$interface}}' Has IP
+  interface_{{$interface}}:
+    title: Interface '{{$interface}}' Has VLAN
     meta:
-      desc: Validates that interface '{{$interface}}' has an IP address
+      desc: Ensures VLAN was created for network interface '{{$interface}}'.
```

Comment:
This test is the IP address test, not the VLAN test, so the previous description was correct.

Reply:
Fixed